mungegithub/pulls/lgtm_after_commit.go

```diff
@@ -65,23 +68,14 @@ func (LGTMAfterCommitMunger) MungePullRequest(client *github.Client, pr *github.
 		return
 	}
 
-	if !HasLabel(issue.Labels, "lgtm") {
+	if !github_util.HasLabel(issue.Labels, "lgtm") {
 		return
 	}
 
 	if lastModified.After(*lgtmTime) {
-		if opts.Dryrun {
-			glog.Infof("Would have removed LGTM label for %d", *pr.Number)
-			return
-		}
 		lgtmRemovedBody := "PR changed after LGTM, removing LGTM."
-		if _, _, err := client.Issues.CreateComment(opts.Org, opts.Project, *pr.Number, &github.IssueComment{Body: &lgtmRemovedBody}); err != nil {
-			glog.Errorf("Error commenting on issue: %v", err)
-			return
-		}
-		if _, err := client.Issues.RemoveLabelForIssue(opts.Org, opts.Project, *pr.Number, "lgtm"); err != nil {
-			glog.Errorf("Error removing 'lgtm': %v", err)
-			return
-		}
+		config.WriteComment(*pr.Number, lgtmRemovedBody)
```

Comment:
Really?  I feel like we'd rather things didn't get merged "wrongly" rather than someone getting mad they didn't get a message about the label being removed...  I'll go your way if you are sure.